src/Sulu/Bundle/WebsiteBundle/Cache/CacheClearer.php

```diff
@@ -69,17 +69,28 @@ public function __construct(
         $this->eventDispatcher = $eventDispatcher;
     }
 
-    public function clear()
+    public function clear(/*?array $tags = []*/)
```

Comment:
would be okay for me to add this without a comment and document it in the `UPGRADE.md` because it will be released in a minor release (not sure what @alexander-schranz thinks about that). but if we add it like this, we should trigger a deprecation if the param is not given. otherwise we will forget about changing it in the next major version